website/docs/d/private_link_endpoint.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+subcategory: ""
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_private_link_endpoint"
+sidebar_current: "docs-azurerm-datasource-private-endpoint"
+description: |-
+  Gets information about an existing Private Link Endpoint
+---
+
+# Data Source: azurerm_private_link_endpoint
+
+Use this data source to access information about an existing Private Link Endpoint.
+
+## Example Usage
+
+```hcl
+data "azurerm_private_link_endpoint" "example" {
+  resource_group_name = "example-rg"
+  name                = "example-private-endpoint"
+}
+
+output "subnet_id" {
+  value = "${data.azurerm_private_link_endpoint.example.id}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the private link endpoint.
+* `resource_group_name` - (Required) The name of the resource group in which the private link endpoint resides.
+
+## Attributes Reference
+
+The following attributes are exported:
```

Comment:
I think the docs are missing the `private_service_connection` block here

Reply:
Removed the data source as it didn't make much sense.